sandbox/tests/test scenes/basis modifiers/26 - normal mapping - metal brdf - point light.appleseed

```diff
@@ -0,0 +1,249 @@
+                <parameter name="filename" value="sources/circle2_DX.png" />
+            </texture>
+            <texture_instance name="normal_inst" texture="normal">
+                <parameter name="addressing_mode" value="wrap" />
+                <parameter name="alpha_mode" value="alpha_channel" />
+                <parameter name="filtering_mode" value="bilinear" />
+                <transform>
+                    <matrix>
+                        1.000000000000000 0.000000000000000 0.000000000000000 0.000000000000000
+                        0.000000000000000 1.000000000000000 0.000000000000000 0.000000000000000
+                        0.000000000000000 0.000000000000000 1.000000000000000 0.000000000000000
+                        0.000000000000000 0.000000000000000 0.000000000000000 1.000000000000000
+                    </matrix>
+                </transform>
+            </texture_instance>
+            <bsdf name="metal_brdf" model="metal_brdf">
+                <parameter name="anisotropy" value="0.0" />
+                <parameter name="edge_tint" value="0.0" />
+                <parameter name="normal_reflectance" value="0.92" />
+                <parameter name="reflectance_multiplier" value="1.0" />
+                <parameter name="roughness" value="0.3" />
+            </bsdf>
+            <material name="metal_mat" model="generic_material">
+                <parameter name="bsdf" value="metal_brdf" />
+                <parameter name="bump_amplitude" value="1.0" />
+                <parameter name="bump_offset" value="0.5" />
+                <parameter name="displacement_map" value="normal_inst" />
+                <parameter name="displacement_method" value="normal" />
+                <parameter name="normal_map_up" value="z" />
+                <parameter name="microfacet_normal_mapping" value="1.0" />
+            </material>
+            <light name="light1" model="point_light">
+                <parameter name="cast_indirect_light" value="true" />
+                <parameter name="exposure" value="0" />
+                <parameter name="importance_multiplier" value="1" />
+                <parameter name="intensity" value="1.0" />
+                <parameter name="intensity_multiplier" value="1" />
+                <transform>
+                    <matrix>
+                        1.000000000000000 0.000000000000000 0.000000000000000 0.000000000000000
+                        0.000000000000000 1.000000000000000 0.000000000000000 0.000000000000000
+                        0.000000000000000 0.000000000000000 1.000000000000000 1.000000000000000
+                        0.000000000000000 0.000000000000000 0.000000000000000 1.000000000000000
+                    </matrix>
+                </transform>
+            </light>
+            <object name="Plane_obj" model="mesh_object">
+                <parameter name="filename" value="_geometry/plane.binarymesh" />
```

Comment:
For all test scenes:
Can you use the `Plane001.binarymesh`, which is already present as well as the `light.obj`?